lib/services/firestore_service.dart

```diff
@@ -1,12 +1,13 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:lost_found_steelhacks/authentication/user.dart';
 import 'package:lost_found_steelhacks/data/item.dart';
 import 'package:rxdart/rxdart.dart';
 
 class FirestoreService {
   //create reference to database
   final lostRef = FirebaseFirestore.instance.collection('lost');
-
   final foundRef = FirebaseFirestore.instance.collection('found');
+  final userRef = FirebaseFirestore.instance.collection('users');
```

Comment:
I don't see this being used anywhere. Do we still need this line?

Reply:
yep I'll remove